docker_xylem/service.py

```diff
@@ -143,7 +143,8 @@ def get_volume(self, request, data):
             return {
                 'Volume': {
                     'Name': name,
-                    'Mountpoint': self.current[name]
+                    'Mountpoint': self.current[name],
+                    'Status': {}
```

Comment:
Don't add trailing commas everywhere, that makes the diff bigger and harder to review. Just add them in places where you're already changing the code.

We can do a general cleanup pass in a separate branch that doesn't make any changes to what the code does, only how it looks.